docs/02-certificate-authority.md

```diff
@@ -54,6 +54,13 @@ chmod +x cfssljson_linux-amd64
 sudo mv cfssljson_linux-amd64 /usr/local/bin/cfssljson
 ```
 
+**Note:** On CoreOS the above directory is read only, therefore you might want to use a different path.
+```
+sudo mkdir -p /opt/bin/
+sudo mv cfssljson_linux-amd64 cfssl_darwin-amd64 /opt/bin/
```

Comment:
Something odd going on. I think, I did this on my Mac when creating the guide. Copying the linux binary as darwin? Needs to be cleaned up to be Darwin or Linux...

Reply:
Ah. Yes. Will fix.